temba/api/views.py

```diff
@@ -1930,11 +1930,14 @@ class FlowRunEndpoint(ListAPIMixin, CreateAPIMixin, BaseAPIView):
     By making a ```GET``` request you can list all the flow runs for your organization, filtering them as needed.  Each
     run has the following attributes:
 
-    * **run** - the id of the run (long) (filterable: ```run``` repeatable)
+    * **uuid** - the UUID of the run (string) (filterable: ```uuid``` repeatable)
     * **flow_uuid** - the UUID of the flow (string) (filterable: ```flow_uuid``` repeatable)
     * **contact** - the UUID of the contact this run applies to (string) filterable: ```contact``` repeatable)
     * **group_uuids** - the UUIDs of any groups this contact is part of (string array, optional) (filterable: ```group_uuids``` repeatable)
     * **created_on** - the datetime when this run was started (datetime) (filterable: ```before``` and ```after```)
+    * **completed** - boolean indicating whether this run has completed the flow (boolean)
+    * **expires_on** - the datetime when this run will expire (datetime) (filterable: ```before``` and ```after```)
```

Comment:
yeah it wasnt immediately clear how/why `created_on` is filterable... so i assumed magic

will remove this bit for `expires_on`